tds-dll-integration-tests/pom.xml

```diff
@@ -79,10 +79,20 @@
 	</dependencies>
 	<build>
 	<plugins>
+		<plugin>
+			<artifactId>maven-surefire-plugin</artifactId>
+			<configuration>
+				<excludes>
+					<exclude>**/*DLL.java</exclude>
+				</excludes>
+				<includes>
+					<include>**/AllTests.java</include>
+				</includes>
+			</configuration>
+		</plugin>
```

Comment:
I configured maven-surefire to only run the AllTests suite (\@Ignored) rather than all of the tests individually.